Source/Blazorise/Components/Dropdown/Dropdown.razor.cs

```diff
@@ -70,11 +70,14 @@ protected override void OnAfterRender( bool firstRender )
                     targetElementId: childrenDropdownToggles?.FirstOrDefault()?.ElementId,
                     altTargetElementId: childrenButtonList?.FirstOrDefault()?.ElementId,
                     menuElementId: childrenDropdownMenus?.FirstOrDefault()?.ElementId,
+                    showElementId: GetShowElementId(),
                     options: new
                     {
                         Direction = GetDropdownDirection().ToString( "g" ),
+                        RightAligned = RightAligned,
                         DropdownToggleClassNames = ClassProvider.DropdownToggle( IsDropdownSubmenu ),
                         DropdownMenuClassNames = ClassProvider.DropdownMenu(),
+                        DropdownShowClassName = GetShowClassName()
```

Comment:
Why is this not part of the ClassProvider? And if it behaves differently for some providers then make use of the BehaviorProvider.

Reply:
Right, I guess I didn't use ClassProvider, cuz it's based on an existing Class Definition. But I'll makle the change.